Assets/Plugins/Source/Editor/Internal/MakefileUtil.cs

```diff
@@ -46,55 +46,25 @@ public bool IsEmpty()
         }
     }
 
-    public class LibraryBuildConfigEditor : IEOSPluginEditorConfigurationSection
+    public class LibraryBuildConfigEditor : ConfigEditor<EOSLibraryBuildConfig>
     {
-        [InitializeOnLoadMethod]
-        static void Register()
+        public LibraryBuildConfigEditor() : base("Platform Library Build Settings",
+            "eos_plugin_library_build_config.json")
         {
-            EOSPluginEditorConfigEditor.AddConfigurationSectionEditor(new LibraryBuildConfigEditor());
         }
 
-        public static string ConfigName = "eos_plugin_library_build_config.json";
-        private EOSConfigFile<EOSLibraryBuildConfig> configFile;
-
-        public void Awake()
-        {
-            var configFilenamePath = EOSPluginEditorConfigEditor.GetConfigPath(ConfigName);
-            configFile = new EOSConfigFile<EOSLibraryBuildConfig>(configFilenamePath);
-        }
-
-        public bool DoesHaveUnsavedChanges()
-        {
-            return false;
-        }
-
-        public string GetNameForMenu()
-        {
-            return "Platform Library Build Settings";
-        }
-
-        public void LoadConfigFromDisk()
-        {
-            configFile.LoadConfigFromDisk();
-        }
```

Comment:
This is another concise example of the functionality that was moved from the most derived class to the base abstract class `ConfigEditor<T>`.